building/windows.md

```diff
@@ -0,0 +1,231 @@
+tty-clock usbutils virt-viewer make autoconf \
+automake cmake libc-client-devel git
+```
+
+  </details>
+  &nbsp;
+
+Before performing the next steps `Cygwin` needs to download all dependencies to the packages that were installed.
+Launch `Cygwin` installation again, in the packages select section we can see tons of libraries were automatically added
+to the list.
+All that we need is to execute this process.
+
+The next step is to prepare the `genext2fs` library and clone the repository. We need to make manually `genext2fs`
+library, because `Cygwin` doesn't support it, and it's mandatory in case of creating images for the ia32 system.
+
+To do this, paste the following into the `Cygwin` terminal:
+
+<details>
+<summary>Make `genext2fs` package and download phoenix-rtos (click to expand)</summary>
+
+``` bash
+git clone --recursive https://github.com/bestouff/genext2fs &&\
+cd genext2fs &&\
+./autogen.sh &&\
+./configure &&\
+make &&\
+make install &&\
+make check &&\
+cd &&\
+git clone --recursive https://github.com/phoenix-rtos/phoenix-rtos-project/
+```
+
+* Folders which contains elements to install can be removed
+
+``` bash
+rm -rf genext2fs apt-cyg
+```
+
+</details>
+&nbsp;
+
+Now, we can follow instructions about building toolchains. Remember to use commands inside
+`phoenix-rtos-project` folder.
+(Quick tip: It is not mandatory but turning off Real-Time Protection in Windows Defender speeds up this process)
+
+``` bash
+ cd phoenix-rtos-project
+```
+
+``` bash
+(cd phoenix-rtos-build/toolchain/ && ./build-toolchain.sh i386-pc-phoenix ~/toolchains/i386-pc-phoenix)
+(cd phoenix-rtos-build/toolchain/ && ./build-toolchain.sh arm-phoenix ~/toolchains/arm-phoenix)
+(cd phoenix-rtos-build/toolchain/ && ./build-toolchain.sh riscv64-phoenix ~/toolchains/riscv64-phoenix)
+(cd phoenix-rtos-build/toolchain/ && ./build-toolchain.sh sparc-phoenix ~/toolchains/sparc-phoenix)
+
+```
+
+Adding toolchain binaries to the PATH variable also can be tricky. On Windows, we move over folders with names
+containing white spaces, after using:
+
+``` bash
+echo "export PATH=$PATH:$HOME/toolchains/i386-pc-phoenix/i386-pc-phoenix/bin/:$HOME/toolchains/arm-phoenix/arm-phoenix/bin/:$HOME/toolchains/riscv64-phoenix/riscv64-phoenix/bin/:$HOME/toolchains/sparc-phoenix/sparc-phoenix/bin/" >> $HOME/.bashrc
+```
+
+After calling this we need to fix paths in `.bashrc` file e.g. `C:\x\y z\d` we must change to `C:\x\'y z'\d`.
+
+This simple change will fix all unwanted messages at
+the start of the `Cygwin` terminal, and it will allow us to use the toolchain without setting the path every time.
+
+To test is recommended to use this build command:
+
+``` bash
+TARGET=ia32-generic-qemu CONSOLE=serial phoenix-rtos-build/build.sh clean core fs image project test
+```
+
+Of course, you can build the system image for this target in different configuration, for example by using the following
+command
+
+```bash
+TARGET=ia32-generic-qemu phoenix-rtos-build/build.sh all
+```

```

Comment:
I think one option to test the building and a reference for more information about the build process would be enough